dbt_sugar/core/documentation/documentation_helper.py

```diff
@@ -0,0 +1,159 @@
+"""Contains documentation related utils."""
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict, List
+
+from dbt_sugar.core.yaml_module.yaml_helpers import open_yaml, save_yaml
+
+COLUMN_NOT_DOCUMENTED = "No description for this column."
+MODEL_NOT_DOCUMENTED = "No description for this model."
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Documentation:
+    """Class to administrate the documentation in DBT."""
+
+    def __init__(self, repository_path: Path) -> None:
+        self.repository_path = repository_path
+        self.dbt_definitions: Dict[str, str] = {}
+        self.save_all_descriptions()
+
+    def get_column_description_from_dbt_definitions(self, column_name: str) -> str:
+        """Searches for the description of a column in all the descriptions in DBT.
+
+        Args:
+            column_name (str): column name to get the description from.
+
+        Returns:
+            str: with the description of the column.
+        """
+        return self.dbt_definitions.get(column_name, COLUMN_NOT_DOCUMENTED)
+
+    def update_description_in_dbt_descriptions(
+        self, column_name: str, column_description: str
+    ) -> None:
+        """Update a column description in the global description dictionary.
```

Comment:
Did we say we would already go for this central dictionary approach?

Reply:
No, we didn't discuss that, but in this PR we are not going for a central dictionary approach.

Reply:
Right I think I got thrown off by the naming/docstrings then. We have a global object but only tat gets created at runtime, do I understand that correctly?